writings/2022-08-23-flashbots-protect-merge-beyond.mdx

```diff
@@ -0,0 +1,51 @@
+---
+slug: flashbots-protect-merge-and-beyond
+title: Flashbots Protect -- The Merge & Beyond
+authors: [angela]
+tags: [protect]
+# image: img/og_forest.jpg
+hide_table_of_contents: false
+---
+
+# Flashbots Protect -- The Merge & Beyond
+
+TL;DR: It has been a great year for Flashbots Protect! Post-merge, all transactions sent to Protect will have frontrunning and revert protection. But due to how mev-boost will be launched at the merge, Flashbots Protect will be down for at least 2 hours after the merge. If you rely on Flashbots Protect as a user or application developer then please plan accordingly.
+
+<!--truncate-->
+
+## Year in Review
+
+In October of last year, Flashbots [launched](https://medium.com/flashbots/announcing-flashbots-protect-9e039e9f0aa3) the public beta of Flashbots Protect, expanding the powerful benefits of bundles to users with just a change in RPC URL in their wallet of choice and to dapp and wallet developers with an easy-to-use SDK.
+
+Since its launch, Flashbots Protect has:
+
+* protected 100,000 IP addresses and 280,000 Ethereum addresses from frontrunning and sandwiches,
```

Comment:
maybe "serviced IP addresses" and "protected 280,000 Ethereum addresses from frontrunning"? you don't really protect IP addresses